Marlin/src/lcd/language/language_pt-br.h

```diff
@@ -319,65 +362,86 @@
 #define MSG_MESH_LEVELING                   _UxGT("Nivelamento da Malha")
 #define MSG_INFO_STATS_MENU                 _UxGT("Estatísticas")
 #define MSG_INFO_BOARD_MENU                 _UxGT("Info. da Placa")
-#define MSG_INFO_THERMISTOR_MENU            _UxGT("Thermistors")
+#define MSG_INFO_THERMISTOR_MENU            _UxGT("Termistores")
 #define MSG_INFO_EXTRUDERS                  _UxGT("Extrusoras")
-#define MSG_INFO_BAUDRATE                   _UxGT("Frequência Baud")
+#define MSG_INFO_BAUDRATE                   _UxGT("Taxa de Transmissão")
 #define MSG_INFO_PROTOCOL                   _UxGT("Protocolo")
-#define MSG_CASE_LIGHT                      _UxGT("Luz da Estrutura")
+#define MSG_CASE_LIGHT                      _UxGT("Luz da Impressora")
 #define MSG_CASE_LIGHT_BRIGHTNESS           _UxGT("Intensidade Brilho")
-#define MSG_INFO_PRINT_COUNT                _UxGT("Qtd de Impressões")
-#define MSG_INFO_COMPLETED_PRINTS           _UxGT("Fim")
-#define MSG_INFO_PRINT_TIME                 _UxGT("Tempo de Impressão")
-#define MSG_INFO_PRINT_LONGEST              _UxGT("Trabalho Mais longo")
-#define MSG_INFO_PRINT_FILAMENT             _UxGT("Total Depositado")
-#define MSG_INFO_MIN_TEMP                   _UxGT("Temp Min")
-#define MSG_INFO_MAX_TEMP                   _UxGT("Temp Max")
+
+#if LCD_WIDTH >= 20
+  #define MSG_INFO_PRINT_COUNT              _UxGT("Quantidade de Impressões")
```

Comment:
This is too long to fit on the display. See if you can get it to 20 characters or less.
```
12345678901234567890|
Quantidade de Impres|sões
```

Reply:
How about: `Total de impressão`?

Reply:
Sorry about that. Total de impressões is okay too.